docs/podman-run.1.md

```diff
@@ -1219,6 +1219,17 @@ Precedence Order:
 
 	   **--env** : Any environment variables specified will override previous settings.
 
+Run containers and set the environment ending with a --*-- and a *****
+
+```
+$ export ENV1=a
+$ $ podman run --env ENV* alpine printenv ENV1
+a
+
+$ podman run --env ENV*****=b alpine printenv ENV*****
+b
```

Comment:
do you need --attach for these two examples also?

Reply:
No
But if I use --attach
```
$ podman run -a --env ENV* alpine printenv ENV*
Error: unable to pull ENV*: error getting default registries to try: invalid reference format
```
I don't know if it's a bug

Reply:
Try `podman run -a --env 'ENV*' alpine printenv | grep ENV` - the shell is interpreting your wildcards

Reply:
yes, I should use grep but don't need --attach
```
$ export ENV1=a
$ export ENV2=b
$ podman run --env ENV* alpine printenv | grep ENV*
ENV=/usr/share/Modules/init/profile.sh
ENV2=b
ENV1=a
$ podman run -a --env ENV* alpine printenv | grep ENV*
Error: unable to pull ENV*: error getting default registries to try: invalid reference format
```